### ISSUE TITLE
SystemError: ffi_prep_closure(): libffi library version

### ISSUE THREAD

**Issue by macdeport** [jbarlow83/OCRmyPDF#823]
**Describe the bug**

libffi library version conflict

**To Reproduce**

`ocrmypdf -l eng --user-words dictionary-fr.txt -k -v1 --pdf-renderer hocr --output-type pdf --jpeg-quality 70 --remove-background --image-dpi 72 --sidecar bid.txt -dr bid.jpg bid.pdf
`
**Expected behavior**

No crash...

**Screenshots**

```
Traceback (most recent call last):
  File "/Users/alain/Library/Python/3.9/bin/ocrmypdf", line 5, in <module>
    from ocrmypdf.__main__ import run
  File "/Users/alain/Library/Python/3.9/lib/python/site-packages/ocrmypdf/__init__.py", line 10, in <module>
    from ocrmypdf import helpers, hocrtransform, leptonica, pdfa, pdfinfo
  File "/Users/alain/Library/Python/3.9/lib/python/site-packages/ocrmypdf/leptonica.py", line 212, in <module>
    def _stderr_handler(cstr):
SystemError: ffi_prep_closure(): bad user_data (it seems that the version of the libffi library seen at runtime is different from the 'ffi.h' file seen at compile-time)

```

**System (please complete the following information):**
 - OS: MacOS
 - Python version: 3.9.7
 - OCRmyPDF version: v12.4.0

**Installation**
How did you install OCRmyPDF? Did you install it from your operating system's
package manager, or using pip?

`pip-3.9 install --user ocrmypdf --upgrade`

**Comment by jbarlow83** [comment]
Are you using Apple Silicon or Intel macOS?

**Comment by macdeport** [comment]
Intel macOS. Thanks.

**Comment by jbarlow83** [comment]
I don't know how to reproduce this issue, unfortunately.

**Comment by macdeport** [comment]
I will try... Thank you.

**Comment by jbarlow83** [comment]
Fixed in ocrmypdf v13.0.0rc1 (note: pre-release) by removing ffi entirely
